understanding/focus-appearance.html

```diff
@@ -53,389 +53,316 @@
                         </li>
                         
                      </ul>
                      
                      
-                     <p>For example, a star-shaped button may use either a focus indicator that follows the
-                        shape of the star or a focus indicator that follows the bounding box of the star.
-                        In the following examples, the same three stars have already been selected, and focus
-                        is on the third star. The first example uses a focus indicator which matches the star
-                        shape of the focused star. The second uses a rectangular indicator.
+                     <p>例えば、星形のボタンは、星の形に沿ってフォーカスインジケータを用いる、又は星の境界ボックスに沿ってフォーカスインジケータを用いてもよい。次の例では、同じ 3 つの星がすでに選択されており、フォーカスは 3 番目の星に存在する。最初の例では、フォーカスしている星の形に合わせたフォーカスインジケータを用いている。2 番目は矩形のインジケータを用いている。
                      </p>
                      
                      
                      <figure id="focus-outline-solid">
-                        <img src="img/focus-indicator-solid-outline.svg" alt="Three of 5 stars are selected with a solid-line focus indicator in the shape of a star outlining the third" />
+                        <img src="img/focus-indicator-solid-outline.svg" alt="五つの星のうち三つが選択され、3 番目の星のアウトラインとして星形の実線のフォーカスインジケータが表示されている。" />
                         
-                        <figcaption>Figure 2 Passes: a solid outline indicator surrounds the third of five stars.</figcaption>
+                        <figcaption>図 2 合格: 実線のアウトラインインジケータが五つの星のうち三つ目を囲んでいる。</figcaption>
                         
                      </figure>
                      
                      
                      <figure id="focus-border-solid">
-                        <img src="img/focus-indicator-solid-border.svg" alt="3 of 5 stars are selected with a solid-line rectangular focus indicator around the third" />
+                        <img src="img/focus-indicator-solid-border.svg" alt="五つの星の三つが選択され、3 番目の星の周りに実線の矩形のフォーカスインジケータが表示されている。" />
                         
-                        <figcaption>Figure 3 Passes: a solidly bound focus rectangle encloses the third of five stars.</figcaption>
+                        <figcaption>図 3 合格: 実線の境界フォーカス矩形が五つの星のうち三つ目を囲んでいる。</figcaption>
                         
                      </figure>
                      
                      
-                     <p>Offsetting indicators slightly from the focused component, as in the examples above,
-                        is not required to meet the minimum area requirement of the success criterion, but
-                        it can help make indicators more visible. In CSS, the <code>outline</code> and <code>outline-offset</code> properties are commonly used to achieve this.
+                     <p>上記の例のように、インジケータをフォーカスコンポーネントから少しずらすことは、達成基準の最小面積の要件を満たすために必要ではないが、インジケータをより見やすくするのに役立つ。CSS では、通例 <code>outline</code> 及び <code>outline-offset</code> プロパティがこれを達成するために使用される。
```

Comment:
実質的には「面積」なのですが、達成基準の翻訳で area を「領域」と訳しているので、minimum area は「最小領域」とします...？

```suggestion
                     <p>上記の例のように、インジケータをフォーカスコンポーネントから少しずらすことは、達成基準の最小領域の要件を満たすために必要ではないが、インジケータをより見やすくするのに役立つ。CSS では、通例 <code>outline</code> 及び <code>outline-offset</code> プロパティがこれを達成するために使用される。
```